casper.py

```diff
@@ -59,6 +59,10 @@ def main():
         '--save', type=bool, default=config.getboolean("Save"),
         help='save the simulation in graphs/ directory'
     )
+    parser.add_argument(
```

Comment:
I might consider changing it to a different name given that full prop is something already. What about "justify-messages" or something?

Only say this b/c Idk how long this hack will be around for :)

Reply:
@djrtwo not sure why this says outdated - but tagging so you see it :)

Reply:
ah yes. changing now